template_lib.py

```diff
@@ -100,12 +100,22 @@ def header(self, text: str, level: int, raw: str) -> str:
         )
 
     def block_code(self, code: str, lang: str) -> str:
+        copyable = False
+        if lang:
+            copyable_s = '#copyable'
+            copyable = lang.endswith(copyable_s)
+            if copyable:
```

Comment:
oh, you shouldn't need this if statment -- can switch to `partition` and it'll do what you want

Reply:
🤦  done